stable/filebeat/values.yaml

```diff
@@ -57,9 +57,21 @@ indexTemplateLoad: []
 plugins: []
   # - kinesis.so
 
-# A map of additional environment variables
-extraVars: {}
-  # test1: "test2"
+# pass custom command. This is equivalent of Entrypoint in docker
+command: []
+
+# pass custom args. This is equivalent of Cmd in docker
+args: []
+
+# A list of additional environment variables
+extraVars: []
+  # - name: TEST1
+  #   value: TEST2
+  # - name: TEST3
+  #   valueFrom:
+  #     configMapKeyRef:
```

Comment:
ah - I see now why you want the list format

Reply:
Yeah, there are values for our deployment that will be loaded from secrets that would be accessed via this manner.